.github/workflows/ci.yml

```
@@ -4,46 +4,105 @@
+          # Gradle 의존성을 JDK와 함께 캐시
+          cache: gradle
 
       - name: Create application.yml file
-        run: echo "${{ secrets.APPLICATION_YML }}" > ./src/main/resources/application.yml
+        run: |
+          mkdir -p ./src/main/resources
+          echo "${{ secrets.APPLICATION_YML }}" > ./src/main/resources/application.yml
+
+      # Gradle Wrapper 검증 (보안)
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@v1
```

actionlint (1.7.7)

the runner of "gradle/wrapper-validation-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)